src/api_routes_v3.rs

```diff
@@ -199,10 +199,11 @@ pub fn config(cfg: &mut ServiceConfig, rate_limit: &RateLimitCell) {
         // Post
         .service(
           // Handle POST to /post separately to add the post() rate limitter
-          resource("/post")
+          scope("/post")
```

Comment:
CI failing. Maybe the fact that there are two `service(scope("/post))` is an issue

Reply:
From what I can tell, this still needs to be resource. Check out the `image` block at the bottom of v4.